taichi/ir/snode.cpp

```diff
@@ -295,6 +295,41 @@ void SNode::end_shared_exp_placement() {
   bit_struct_type_builder->end_placing_shared_exponent();
 }
 
+void SNode::lazy_grad() {
+  make_lazy_place(this, snode_to_glb_var_exprs_,
+                  [&](std::unique_ptr<SNode> &c, std::vector<Expr> &new_grads,
```

Comment:
instead of using `[&]` can you explicitly list out things we wanna reference or copy? ;) Not sure if I'm missing anything, but this list doesn't seem long so it'd be nice to be explicit

Reply:
Have explicitly capture this here.